### PR TITLE
Return of the Maxim, plus some tweaks

### PR DESCRIPTION
**Pull request by zombiemimic**
## About The Pull Request
Deep in the code, the Maxim was buried. WIth this, I bring it back from the dead, along with some other stuff.
tachanka
image

## Why It's Good For The Game
Gives excels a new gun

## Changelog
:cl:
add: Added the Maxim machine gun to excel roster
tweak: tweaked mosin disk rarity
tweak: changed excel turret as the maxim on a stand that it is
code: added slowdown_hold for slowdown caused by holding items
/:cl:

### CODE REVIEW

**Comment by Jan-Blasiak** [comment]
Am I reading this right? It doesn't have a slowdown?

**Comment by zombiemimic** [comment]
> Am I reading this right? It doesn't have a slowdown?

It's a new variable called slowdown_hold

File: code/modules/mob/living/carbon/human/human_movement.dm [hunk 1]
```diff
@@ -47,6 +47,12 @@
 
 	if(slowdown)
 		tally += 1
+	
+	if(l_hand.slowdown_hold)
```
Comment (MLGTASTICa) [inline code/modules/mob/living/carbon/human/human_movement.dm:51]: you can just remove the if's and just do tally += l_hand.slowdown_hold + r_hand.slowdown_hold , they're just needless bloat

File: code/game/machinery/excelsior/ex_turret.dm [hunk 2]
```diff
@@ -13,7 +13,7 @@
 	installation = null
 	var/obj/item/ammo_magazine/ammo_box = /obj/item/ammo_magazine/ammobox/lrifle
 	var/ammo = 0 // number of bullets left.
-	var/ammo_max = 160
+	var/ammo_max = 96
```
Comment (MLGTASTICa) [inline code/game/machinery/excelsior/ex_turret.dm:16]: undocumented change .

File: code/game/objects/items/weapons/circuitboards/machinery/excelsior.dm [hunk 3]
```diff
@@ -76,7 +76,7 @@
 	board_type = "machine"
 	origin_tech = list(TECH_COMBAT = 3, TECH_COVERT = 2)
 	req_components = list(
-		/obj/item/weapon/stock_parts/manipulator = 2,
+		/obj/item/weapon/gun/projectile/automatic/maxim = 1,
```
Comment (MLGTASTICa) [inline code/game/objects/items/weapons/circuitboards/machinery/excelsior.dm:79]: undocumented change

File: code/game/objects/items/weapons/design_disks/excelsior.dm [hunk 4]
```diff
@@ -130,8 +133,8 @@
 	it is their wisdom which has brought the great work to bear - \
 	with tools such as these, their great vision was brought to light.\""}
 	icon_state = "excelsior"
-	spawn_tags = SPAWN_TAG_DESIGN_ADVANCED_EXCELSIOR
-	rarity_value = 50
+	spawn_tags = SPAWN_TAG_DESIGN_COMMON_EXCELSIOR
```
Comment (MLGTASTICa) [inline code/game/objects/items/weapons/design_disks/excelsior.dm:136]: undocumented change

Comment (zombiemimic) [reply code/game/objects/items/weapons/design_disks/excelsior.dm:136]: no, that's documented

Comment (MLGTASTICa) [reply code/game/objects/items/weapons/design_disks/excelsior.dm:136]: my bad , looked at the wrong part.

File: code/modules/projectiles/guns/projectile/automatic/maxim.dm [hunk 5]
```diff
@@ -16,29 +16,38 @@
...
 	reload_sound = 'sound/weapons/guns/interact/lmg_magin.ogg'
 	cocked_sound = 'sound/weapons/guns/interact/lmg_cock.ogg'
 	fire_sound = 'sound/weapons/guns/fire/lmg_fire.ogg'
-	recoil_buildup = 3
-	one_hand_penalty = 30 //not like it's used anyway, but LMG level
+	recoil_buildup = 2.2
+	one_hand_penalty = 45 //not like it's used anyway, but HMG level
 	init_firemodes = list(
-		FULL_AUTO_600,
+		list(mode_name = "full auto",  mode_desc = "600 rounds per minute",   mode_type = /datum/firemode/automatic, fire_delay = 1, icon="auto", damage_mult_add = -0.1, move_delay=6),
```
[the remark this answers is not in the document]
Comment (zombiemimic) [reply code/modules/projectiles/guns/projectile/automatic/maxim.dm:28]: for the move delay

File: code/modules/projectiles/guns/projectile/automatic/maxim.dm [hunk 6]
```diff
@@ -16,29 +16,38 @@
...
 		list(mode_name="short bursts", mode_desc="dakka", burst=5,    burst_delay=1, move_delay=6,  icon="burst"),
 		list(mode_name="long bursts", mode_desc="Dakka", burst=8, burst_delay=1, move_delay=8,  icon="burst"),
-		list(mode_name="suppressing fire", mode_desc="DAKKA", burst=16, burst_delay=1, move_delay=11,  icon="burst")
+		list(mode_name="suppressing fire", mode_desc="DAKKA", burst=16, burst_delay=1, move_delay=110,  icon="burst")
 		)
 	twohanded = TRUE
 	spawn_blacklisted = TRUE
+	slowdown_hold = 5
```
[the remark this answers is not in the document]
Comment (zombiemimic) [reply code/modules/projectiles/guns/projectile/automatic/maxim.dm:35]: it is, by design

**Comment by drexample** [comment]
based and balancepilled, let's see if players like it

**Comment by Medcar** [comment]
local man breaks the fabric of reality itself

**Pull request merged**